Vic2ToHoI4/Source/HOI4World/HoI4World.cpp

```diff
@@ -1669,3 +1682,77 @@ closedProvinces.insert(make_pair(thisAdjacency.to, thisAdjacency.to));
 return -1;
 
 }*/
+
+void HoI4::World::setSphereLeaders()
+{
+	for (auto GP: greatPowers)
+	{
+		for (auto& relationItr: GP->getRelations())
+		{
+			if (relationItr.second.getSphereLeader())
+			{
+				auto sphereling = findCountry(relationItr.first);
+				sphereling->setSphereLeader(GP->getTag());
```

Comment:
You should account for when the sphereling isn't found:

```if (sphereling)```
```{```
```  sphereling->setSphereLeader(GP->getTag());```
```}```

Reply:
Actually, getSphereLeader() is `bool`. It checks if sphereLeader is set to `true` in the relations object.

Reply:
Ah, all good then!